src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/XamarinProject.cs

```diff
@@ -157,6 +161,15 @@ public virtual List<ProjectResource> Save (bool saveProject = true)
 				});
 			}
 
+			Func<Import,ProjectResource> create_import_reference = import => new ProjectResource () {
```

Comment:
@jonpryor should this be a local function? Do we prefer that other a normal method now?